delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -116,43 +117,54 @@ class AclsRoutes(identities: Identities, acls: Acls)(implicit
     pathPrefix("acls") {
       extractCaller { implicit caller =>
         concat(
+          // SSE realms
+          (pathPrefix("events") & pathEndOrSingleSlash) {
+            operationName(s"$prefixSegment/acls/events") {
+              lastEventId { offset =>
+                completeStream(acls.events(offset))
+              }
+            }
+          },
           extractAclAddress { address =>
             extractSubject { implicit subject =>
               parameter("rev" ? 0L) { rev =>
                 concat(
                   (put & entity(as[AclInput])) { aclEntity =>
-                    authorizeFor(address, aclsWrite).apply {
-                      val status = if (rev == 0L) Created else OK
-                      completeIO(status, acls.replace(address, aclEntity.toAcl, rev).map(_.void))
+                    operationName(s"$prefixSegment/acls/replace") {
```

Comment:
I would recommend that we keep the operationName to match the uri with escaped addresses, like so:
```
operationName(s"$prefixSegment/acls/{}/{}")
or
operationName(s"$prefixSegment/acls/{}")
or
operationName(s"$prefixSegment/acls")
```
replace or append looks like an RPC api, so the use of kamon tags would be more appropriate, like `"operation" -> "replace"`